niworkflows/interfaces/surf.py

```diff
@@ -131,6 +132,15 @@ class GiftiNameSource(SimpleInterface):
     >>> res.outputs.out_name
     'space-fsaverage.R.func'
 
+    >>> namer = GiftiNameSource()
+    >>> namer.inputs.pattern = r'(?P<LR>[lr])h.(?P<space>\w+).gii'
+    >>> namer.inputs.template = r'space-{space}.{den}.{LR}.func'
```

Comment:
```suggestion
    >>> namer.inputs.template = r'space-{space}_den-{density}_hemi-{hemi}.func'
```

Reply:
this would require a fair bit of structuring on fmriprep's side - mainly hitting all fsaverage surfaces

Reply:
but you can redefine the input of the nipype interface right?

Reply:
oh, i thought this comment was on the fmriprep PR.

changing `den` to `density` sounds reasonable, but BIDS-ifying the whole thing seems unnecessary, at least for these changes

Reply:
Yeah - since this is going to show up in the documentation as an example, I'd go with a BIDS-y example.